packages/ui-react/src/components/Shelf/Shelf.module.scss

```diff
@@ -38,6 +34,15 @@
   text-overflow: ellipsis;
 }
 
+.slider {
+  position: static;
+  overflow: visible;
+
+  :global(.TileSlider-list) >li > a {
+    white-space: initial;
```

Comment:
Interesting trick. I have no idea what this `white-space: initial;` solves. Can you explain?